includes/admin/settings.php

```diff
@@ -87,6 +91,43 @@ public function column_payment_method_value( $value, $affiliate ) {
 		return $value;
 	}
 
+	/**
+	 * Adds a "Payment Method" column to the referrals screen.
+	 * 
+	 * @since 2.3.3
```

Comment:
```suggestion
	 * @since 2.3.4
```
Bump version to 2.3.4